src/PhpSpec/Runner/Maintainer/CollaboratorsMaintainer.php

```diff
@@ -201,7 +201,7 @@ private function getParameterTypeFromIndex(\ReflectionClass $classRefl, \Reflect
     private function getParameterTypeFromReflection(\ReflectionParameter $parameter): string
     {
         try {
-            if (null === $class = $parameter->getClass()) {
+            if (null === $class = $parameter->getType()) {
```

Comment:
This is actually not the same as before. A reflection exception will not be thrown if the class doesn't exist. You need to do a class_exists check and throw the collaborator not found yourself.

Reply:
oh that explains why that exception is caught elsewhere too - thanks

Reply:
> A reflection exception will not be thrown if the class doesn't exist.

By that I mean, `getClass` would throw an exception, but `getType` does not. :)